modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -95,3 +112,88 @@
 	cost = 12
 	surplus = 0
 	exclude_modes = list(/datum/game_mode/nuclear/clown_ops)
+
+//LOADOUTS
+
+/datum/uplink_item/loadout_skyrat
+	category = "Loadout"
+	surplus = 0
+	cant_discount = TRUE //I honestly don't think discount is worth it for those things, sorry.
+
+/datum/uplink_item/loadout_skyrat/recon
+	name = "Reconnaisance bundle"
+	desc = "Get in and get out as quickly as you came with this unique kit of gear specialized in infiltration and observation."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/recon
+	cost = 25 // I don't think this is enough TCs
+
+/datum/uplink_item/loadout_skyrat/spy
+	name = "Spy bundle"
+	desc = "Blend into the environment or any of crowd with this state-of-the-art stealth kit, perfect for infiltration experts."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/spy
+	cost = 25
+
+/datum/uplink_item/loadout_skyrat/stealthop
+	name = "Burglar Bundle"
+	desc = "Not a thing aboard the station is safe from your grubby hands with this specialized set of gear, perfect for the enterprising thief."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/stealthop
+	cost = 25
+
+/datum/uplink_item/loadout_skyrat/hacker
+	name = "Hacker bundle"
+	desc = "Subvert everything in sight using some of the most advanced tools available to operatives. If it’s powered, it’s already under your thumb."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/hacker
+	cost = 25
+
+/datum/uplink_item/loadout_skyrat/sniper
+	name = "Hitman bundle"
+	desc = "Carry out efficient and likely not-so-discreet assassinations with this unique sniping kit. Carrying case not included."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/sniper
+	cost = 25
+
+/datum/uplink_item/loadout_skyrat/metaops
+	name = "Bulldog Operative bundle"
+	desc = "Fight the power with this frontline combatant kit, featuring armor and armaments commonly utilized by assault operative teams."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/metaops
+	cost = 25
```

Comment:
```suggestion
	cost = 28
```
**This is the minimum acceptable TC cost, assuming it's even acceptable to begin with.**
Bulldog = 15
Syndie Hardsuit = 8 (23 total)
Emag = 4 (27 total)
Airlock Emag = 3 (30 total)
C4 = 1 (31 total)
C4 = 1 (32 total)
Microbomb Implant = 2 (34 total)
Microbomb Implant if we were to price it for traitors = 5-ish (37-ish)

We also wouldn't give the microbomb to traitors for merely 2TC. I'm a little concerned on offering a "haahaa gib myself" option to traitors too, but I'll put that concern aside for now.

Reply:
This kit is actually a direct copy-and-paste of the Meta-Ops random kit you can currently get ingame for 20tc. While I understand limiting the overall costs of kits here as they aren't random, why haven't we thus changed how the random kits work?